ci/test_suites/authenticators_k8s/dev/Dockerfile.test

```diff
@@ -1,13 +1,8 @@
-FROM google/cloud-sdk:388.0.0
+FROM google/cloud-sdk:421.0.0
```

Comment:
I know Snyk suggested 421. Was there additional reason not to just go all the way to 424 (the latest)? If not, might as well go to the latest.